owl/EASE-OBJ.owl

```diff
@@ -455,6 +666,54 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Barrier -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Barrier">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Excluder"/>
+        <rdfs:comment>A tool used as a barrier to prevent access to a protected space, or to prevent enclosed objects from escaping.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Blockage -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Blockage">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Exclusion"/>
```

Comment:
As before, while blockages are often used for exclusion, that's not always the case.

Reply:
The blocker prevents to move on certain pathways, but it only excludes objects that can not overcome the blockage -- e.g. you won't prevent dust particles from moving around there.